### PR TITLE
EBF-208: Design System Cards

### PR DESCRIPTION
**Pull request by moysa**
- Implement Primary Card with and without image and content
- Implement Options Card
- Implement Price component

### CODE REVIEW

File: addon/components/price/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+export default class PriceComponent extends Component {
+  get formattedPrice() {
+    const { currency, locale, amount } = this.args;
+
+    const formatter = new Intl.NumberFormat(locale || 'de-DE', {
+      style: 'currency', currency: currency || 'EUR',
+    });
+
+    return formatter.format(amount);
+  }
```
Comment (obyknovenius) [inline addon/components/price/index.js:19]: Could this be a helper?

File: addon/components/card/option.hbs [hunk 2]
```diff
@@ -1,17 +1,7 @@
...
-        description=(component "card/description")
-      )}}
-    {{else}}
-      <Card::Title @title={{@title}} />
-      <Card::Description @value={{@description}} />
-    {{/if}}
-  </div>
-</div>
\ No newline at end of file
+<Card::Base class="card-option card-select {{if @selected 'active'}}"
```
Comment (obyknovenius) [inline addon/components/card/option.hbs:1]: Should it have `card-hover` class? Like [this example](https://ebf.github.io/ebf-design-system/versions/master/examples/esim-onboarding-multisim).

**Review by obyknovenius — APPROVED** [review]

File: addon/components/card/option.hbs (hunk 2 above)
Comment (moysa) [reply addon/components/card/option.hbs:1]: Yup. Thanks

File: addon/components/price/index.js (hunk 1 above)
Comment (moysa) [reply addon/components/price/index.js:19]: Good point!

**Pull request merged**